src/main/java/uk/gov/hmcts/ccd/domain/types/sanitiser/client/DocumentManagementRestClient.java

```diff
@@ -43,18 +38,6 @@ public Document getDocument(FieldTypeDefinition fieldTypeDefinition, String url)
         final HttpHeaders headers = securityUtils.authorizationHeaders();
         headers.setContentType(MediaType.APPLICATION_JSON);
         final HttpEntity requestEntity = new HttpEntity(headers);
-
-        // Need to ignore the response Content Type header from Document Management because it is not the expected type
-        // of "application/json". See https://stackoverflow.com/a/44219832 for the same solution to a similar problem.
-        List<HttpMessageConverter<?>> messageConverters = new ArrayList<>();
-        //Add the Jackson Message converter
-        MappingJackson2HttpMessageConverter converter = new MappingJackson2HttpMessageConverter();
-        // Note: here we are making this converter to process any kind of response,
-        // not only application/*json, which is the default behaviour
-        converter.setSupportedMediaTypes(Collections.singletonList(MediaType.ALL));
-        messageConverters.add(converter);
-        restTemplate.setMessageConverters(messageConverters);
-
         Document document = null;
         try {
```

Comment:
I liked the suggestion in the ticket to change the response code from 400 to 500. Shall we do that?